datasetComparison/src/test/scala/za/co/absa/hermes/datasetComparison/DatasetComparisonConfigSuite.scala

```diff
@@ -14,21 +14,35 @@
  * limitations under the License.
  */
 
-package za.co.absa.hermes.datasetComparison.config
+package za.co.absa.hermes.datasetComparison
 
 import org.scalatest.FunSuite
 
-class TypeSafeConfigSuite extends FunSuite {
+class DatasetComparisonConfigSuite extends FunSuite {
+  test("Manual Config Correct") {
+    val conf = DatasetComparisonConfig("errCol", "_actual", "_expected", false)
+    assert(conf.validate().isSuccess)
+    assert("errCol" == conf.errorColumnName)
+    assert("_actual" == conf.actualPrefix)
+    assert("_expected" == conf.expectedPrefix)
+    assert(!conf.allowDuplicates)
+  }
+
+  test("Manual Config Bad Column Name") {
+    val conf = DatasetComparisonConfig("errCol", "_actua l", "_expected", false)
+    assert(conf.validate().isFailure)
+  }
+
   test("Default Config Loaded") {
-    val conf = new TypesafeConfig(None)
+    val conf = DatasetComparisonConfig.default
     assert("errCol" == conf.errorColumnName)
     assert("actual" == conf.actualPrefix)
     assert("expected" == conf.expectedPrefix)
```

Comment:
I know we have discussed this already before, but I still find this suboptimal. Sorry.

```suggestion
    assert(conf.errorColumnName == "errCol")
    assert(conf.actualPrefix == "actual")
    assert(conf.expectedPrefix == "expected")
```

diverging from the conventional order results in incorrect scala-test based messages. E.g.:
```scala
assert("expectedValue" == "actualValue")
```
will generate output:
```
"[expected]Value" did not equal "[actual]Value"
ScalaTestFailureLocation: za.co.absa.hermes.datasetComparison.DatasetComparisonConfigSuite$$anonfun$1 at (DatasetComparisonConfigSuite.scala:25)
Expected :"[actual]Value"
Actual   :"[expected]Value"
```
This results in incorrect error messages and possible confusion of the reader.

(this issue appears elsewhere in the code, too)

Reply:
It's okay, I will switch it. It is just hard to kick the habit 👍